UpdateApplication.ps1

```diff
@@ -0,0 +1,120 @@
+[cmdletbinding()]
+Param
+(
+    [Parameter(ParameterSetName = 'Customize')]	
+    [String]
+    $webApplicationId,
+    [Parameter(ParameterSetName = 'Customize')]
+    [int]
+    $timeoutMin = 5,
+    [Parameter(ParameterSetName = 'Customize')]
+    [int]
+    $httpPort = 19080,
+    [Parameter(ParameterSetName = 'Customize')]
+    [string]
+    $logFile
+)
+
+
+$headers = $null
+. "$PSScriptRoot\Common.ps1"
+$graphAPIFormat = $resourceUrl + "/v1.0/{0}"
+$global:ConfigObj = @{}
+
+function main () {
```

Comment:
>main [](http://example.com/codeflow?start=9&length=4)

I'd suggest a 'whatIf' mode where the diffs are calculated but not applied..

Reply:
Added a whatIf check